nncf/experimental/post_training/compression_builder.py

```diff
@@ -42,23 +45,41 @@ def add_algorithm(self, algorithm: Algorithm) -> None:
         self.algorithms.append(algorithm)
 
     def _create_engine(self, backend: BackendType) -> Engine:
-        # TODO (Nikita Malinin): Place "ifs" into the backend-specific expandable structure
+        """
+        Creates backend-specific Engine.
+        """
         if backend == BackendType.ONNX:
-            from nncf.experimental.onnx.engine import ONNXEngine  # pylint: disable=cyclic-import
+            from nncf.experimental.onnx.engine import ONNXEngine
             return ONNXEngine()
         return None
 
-    def _create_statistics_aggregator(self, engine: Engine, dataset: Dataset, backend: BackendType):
+    def _create_statistics_aggregator(self,
+                                      engine: Engine,
+                                      dataset: Dataset,
+                                      model_transformer: StaticModelTransformerBase,
+                                      backend: BackendType) -> StatisticsAggregator:
+        """
+        Creates backend-specific StatisticsAggregator.
```

Comment:
Add parameter and return type descriptions to the docstring according to the common format.

Reply:
Done